draft-ietf-quic-transport.md

```diff
@@ -832,6 +832,27 @@ Use of a secure random number generator {{?RFC4086}} is not necessary for
 generating the initial packet number, nor is it necessary that the value be
 uniformly distributed.
 
+### New Connection ID Packet Numbers
+
+Packets with with new connection IDs have special rules for encoding the packet
+number in the headers.  They use an offset (see {{packet-number-offset}}) to
+transform the packet number via the following:
```

Comment:
Why does this only apply to new connection IDs? Wouldn't it be easier to apply this transformation to every packet number?

Reply:
The gap in the actual packet number was introduced at the point you sent the first packet with a new CID.  This proposal is a transformation of the actual packet number for wire presentation, so it needs to occur every time.

Reply:
I think that's a clear mistake, the transformation does apply to all packet numbers on the network